src/subscript/sunsch/sunsch.py

```diff
@@ -724,36 +563,28 @@ def main():
     if args.dategrid:
         cli_config["dategrid"] = args.dategrid
 
-    # Merge defaults-, yaml- and command line options, and then validate:
-    config = configsuite.ConfigSuite(
-        {},
-        get_schema(),
-        layers=(defaults_config, yaml_config, cli_config),
-        deduce_required=True,
-    )
-    if not config.valid:
-        logger.error(config.errors)
-        logger.error("Your configuration is invalid. Exiting.")
-        sys.exit(1)
+    merged_config = defaults_config.copy()
+    merged_config.update(yaml_config)
+    merged_config.update(cli_config)
 
-    if args.verbose and config.snapshot.output != __MAGIC_STDOUT__:
+    config = SunschConfig(**merged_config)
```

Comment:
Style: Could write something like this:

```
SunschConfig(**{
    "output": "-", 
    "startdate": datetime.date(1900, 1, 1),
    **yaml_config,
    **cli_config
})
```

instead of defining a default_config

Reply:
thanks, will try

Reply:
Pydantic did not like merging these settings; 
`E       TypeError: ModelMetaclass object got multiple values for keyword argument 'output'`